ci/pipelines/geode-build/jinja.template.yml

```diff
@@ -92,8 +92,10 @@ groups:
   jobs:
   - {{ build_test.name }}
   {{- all_gating_jobs() | indent(2) }}
-  {%- if repository.sanitized_fork == repository.upstream_fork %}
+  {%- if repository.sanitized_fork == "apache" and repository.branch == "develop" %}
   - UpdatePassingRef
+  {%- endif %}
+  {%- if repository.upstream_fork != "apache" or repository.branch == "develop" %}
```

Comment:
So we will only not `PublishArtifacts` if we are on `apache` on a branch that is not `develop`. Seems reasonable enough to assume that's a release branch. Do we ever build with a repo that is not `apache`?

Reply:
Downstream consumers of the pipeline may publish their releases without the manual process Apache Geode goes through.